index.js

```diff
@@ -1,54 +1,101 @@
-// selecting elements
-const titleInput = document.querySelector('#title');
-const authorInput = document.querySelector('#author');
-const bookList = document.querySelector('#book-list');
-const formGroup = document.querySelector('#form-container');
-let addOn;
-let bookObj = JSON.parse(localStorage.getItem('saveItem')) || [];
-
-// Add book
-function newAdd() {
-  addOn = {
-    title: titleInput.value,
-    author: authorInput.value,
-    id: Math.floor(Math.random() * 1000000),
-  };
-  bookObj.push(addOn);
-  localStorage.setItem('saveItem', JSON.stringify(bookObj));
+/* eslint-disable */
+// Book class
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
```

Comment:
- [OPTIONAL] I think it would be better to use one class containing all the methods you want instead of three classes 💡